### ISSUE TITLE
Pre-trained weights link leads to 404

### ISSUE THREAD

**Issue by baoruxiao** [scheckmedia/keras-shufflenet#4]
Hi Tobias,

I tried to download your weights file but got 404 page.
Would you mind checking it out?

Appreciate it!

**Comment by baoruxiao** [comment]
I see the weights in folder, sorry to bother.

**Comment by scheckmedia** [comment]
Anyway I've to fix the URL in the readme. Thanks for the advice!
